README.md

```diff
@@ -2,57 +2,317 @@
-communications, and other user-defined fields (such as memory
-usage). Documentation on the JSON format can be found [in vt's
-documentation](https://darma-tasking.github.io/docs/html/node-lb-data.html)
-and the JSON schema validator is located [in the
-repository](https://github.com/DARMA-tasking/vt/blob/develop/scripts/JSON_data_files_validator.py).
-
-The task visualizer, using this input, produces Exodus meshes to
+`vt-tv` provides visualizations of an application's work-to-rank mappings, communications, and memory usage.
+
+Jump to:
+- [Overview](#overview)
+- [Getting Started](#getting-started)
+- [Installation and Usage](#installation-and-usage)
+- [Design Information](#design-information)
+
+## Overview
+
+Specifically, the task visualizer takes in JSON files that describe work as a series of phases and subphases
+that contain 1) tasks for each rank, 2) communications, and 3) other user-defined fields (such as memory usage).
+
+Using this input data, the task visualizer produces Exodus meshes to
 describe the ranks and objects over time, which can be visualized
 using Paraview. Additionally, the task visualizer can produce PNGs
 directly using a VTK workflow to render a visualization of ranks and
-tasks over phases.
+tasks over phases (as seen below).
 
 ![Example Output PNG](./docs/example-output-image.png)
 
-## Building the Python bindings
+## Getting Started
 
-### Requirements
+You will need the following dependencies:
 
-In order to build the python bindings, make sure you have a Python <ins>`3.8` or `3.9`</ins> environment, with the `nanobind` package installed. You can install `nanobind` with `pip`:
+1. A C++ compiler that supports C++17
+2. [`cmake`](https://cmake.org/cmake/help/latest/) >= 3.17
+3. [`VTK`](https://docs.vtk.org/en/latest/index.html) (build instructions [here](https://gitlab.kitware.com/vtk/vtk/-/blob/master/Documentation/docs/build_instructions/build.md))
 
-```bash
-pip install nanobind
+Begin by cloning `vt-tv`:
+
+```
+git clone https://github.com/DARMA-tasking/vt-tv.git
+```
+
+_In future directions, we will assume that the `vt-tv` source is located in `${VTTV_SOURCE_DIR}`._
+
+## Installation and Usage
+
+`vt-tv` can installed as either a standalone C++ app or as a Python module. Instructions for both cases are included in the dropdowns below.
+
+<details>
+<summary><b>Standalone</b></summary>
+
+### 1. Build
+
+For the simplest build, run from `${VTTV_SOURCE_DIR}`:
+
+```
+VTK_DIR=/path/to/vtk/build ./build.sh
+```
+
+To build and run tests, add the `--tests-run` flag:
+
+```
+VTK_DIR=/path/to/vtk/build ./build.sh --tests-run
+```
+
+More documentation for `build.sh` can be found within the script itself, including examples.
+
+Alternatively, for an interactive build process, run:
+
+```
+./interactive_build.sh
 ```
 
-You must have a C++ compiler that supports C++17, and `cmake` >= 3.17.
+_In future directions, we will assume  that the `vt-tv` build is in `${VTTV_BUILD_DIR}`._
```

Comment:
What do you mean by `In future directions` here? `From now onwards`?

Reply:
Yes, I've changed this to `From now on`